src/product.cc

```diff
@@ -78,6 +86,7 @@ std::string Product::package_name() {
   return package_name_;
 }
 
+// Not sure what to do here with the unit value and packaging...
```

Comment:
This should behave the same way as `Extract` from the point if view of the unit value

Reply:
But I think I'll need to remember why this extract needs to be special and add better comments for why